tools/azure-sdk-tools/devtools_testutils/resource_testcase.py

```diff
@@ -97,18 +108,12 @@ def remove_resource(self, name, **kwargs):
                     azure_poller.wait(kwargs.get("wait_timeout"))
                     if azure_poller.done():
                         return
-                    raise AzureTestError(
-                        "Timed out waiting for resource group to be deleted."
-                    )
+                    raise AzureTestError("Timed out waiting for resource group to be deleted.")
                 else:
                     self.client.resource_groups.delete(name, polling=False)
```

Comment:
Do we know how reliable this runs? Is there any fallback that helps ensure this delete runs?